apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v1/eobByPatientIdAll.json

```diff
@@ -2070,14 +2070,6 @@
           } ]
         } ]
       }, {
-        "extension" : [ {
```

Comment:
This goes for this set of removed lines as well as for all others. So I understand baded on the new logic that these should be removed, but shouldn't we attempt to modify our test data to assert that the logic changes work and that they generate .json ?

Reply:
All fixed now. Youre right. JSON now reflects dynamically generated diagnosis codes.